vaadin-select-flow/pom.xml

```diff
@@ -6,14 +6,20 @@
     <parent>
         <groupId>com.vaadin</groupId>
         <artifactId>vaadin-select-flow-parent</artifactId>
-        <version>2.1-SNAPSHOT</version>
+        <version>2.2-SNAPSHOT</version>
     </parent>
 
     <artifactId>vaadin-select-flow</artifactId>
     <packaging>jar</packaging>
 
     <name>Vaadin Select</name>
 
+    <repositories>
+        <repository>
+            <id>webjars-repo</id>
+            <url>https://dl.bintray.com/webjars/maven/</url>
+        </repository>
+    </repositories>
```

Comment:
We shouldn't add bintray repository, if the webjar is not available in maven central we should try to generate it again via webjars.org

Reply:
Done